backend/lcfs/web/api/forms/views.py

```diff
@@ -0,0 +1,114 @@
+# TODO: Example APIs! Replace with actual business logic.
+
+import structlog
+from typing import Optional
+
+from fastapi import APIRouter, Request, HTTPException, Depends
+from pydantic import BaseModel
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+
+from lcfs.web.core.link_key_auth import anonymous_form_handler
+
+from lcfs.db.models.user.Role import RoleEnum
+from lcfs.db.dependencies import get_async_db_session
+from lcfs.db.models.form.Form import Form
+
+logger = structlog.get_logger(__name__)
+router = APIRouter()
+
+
+# Schemas
+class FormResponse(BaseModel):
+    form_id: int
+    name: str
+    slug: str
+    description: Optional[str] = None
+    organization_name: Optional[str] = None
+    status: str
+    message: str
+
+
+@router.get("/{form_slug}")
+@anonymous_form_handler(
+    allowed_roles=None,  # Allow any authenticated user
+    allow_link_key_access=False,
+    require_authenticated_user=True,
+)
+async def get_form_by_slug_authenticated(
+    request: Request,
+    form_slug: str,
+    db: AsyncSession = Depends(get_async_db_session),
+):
+    """
+    Get form by slug for authenticated users (BCeID users)
+    """
+    logger.info(
```

Comment:
don't think we need this logger here. Error logging is already handled, this would be extraneous.